components/Modal/modal.css

```diff
@@ -0,0 +1,29 @@
+.modal-footer {
+  justify-content: end;
+  margin: 2rem 0rem 1rem 0rem;
+}
+.modal-active {
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  background-color: var(--color-modal-bg);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.dialog {
+  background-color: var(--color-grey-light-1);
+  position: relative;
+  max-width: 50rem;
+  width: 100%;
+  height: 50vh;
+  overflow: auto;
+  display: flex;
+  padding: 15px 30px;
```

Comment:
Better to use rems in place of px if possible

Reply:
thanks, I forgot to replace I think. will update.